src/components/Match/Overview/PicksBans.jsx

```diff
@@ -70,21 +66,25 @@ img {
 }
 `;
 
-const PicksBans = ({ data, strings, style }) => (
-  <Styled style={style}>
-    <div className="PicksBans">
-      {data.map(pb => (
-        <section key={pb.order}>
-          <HeroImage id={pb.hero_id} imageSizeSuffix={IMAGESIZE_ENUM.SMALL.suffix} data-isPick={pb.is_pick} />
-          {!pb.is_pick && <div className="ban" />}
-          <aside>
-            {pb.is_pick ? strings.match_pick : strings.match_ban} <b>{pb.order + 1}</b>
-          </aside>
-        </section>
-      ))}
-    </div>
-  </Styled>
-);
+
+const PicksBans = ({ gameMode, data, strings, style }) => {
+  let banNumber = 0;
+  return (
+    <Styled style={style}>
+      <div className="PicksBans">
+        {data.map(pb => (
+          <section key={pb.order}>
+            <HeroImage id={pb.hero_id} imageSizeSuffix={IMAGESIZE_ENUM.SMALL.suffix} data-isPick={pb.is_pick} />
+            {!pb.is_pick && <div className="ban" />}
+            <aside>
+              {pb.is_pick ? strings.match_pick : strings.match_ban} <b>{gameMode === 22 ? pb.is_pick ? pb.order + 1 : banNumber+=1 : pb.order + 1}</b>
```

Comment:
can we avoid mutations in this rendering code? It can be confusing.

Reply:
[82055b6](https://github.com/odota/web/pull/2889/commits/82055b6d3983922aa3d09f0149b7d944ecfeb9d3) seems to work.

PS Hit some issues merging causing extra commits,  please lmk if theres anything else